rest-api-spec/src/main/resources/rest-api-spec/api/query_ruleset.list.json

```
@@ -0,0 +1,36 @@
+{
+  "query_ruleset.list": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/list-query-rulesets.html",
+      "description": "Lists query rulesets."
+    },
+    "stability": "experimental",
+    "visibility": "feature_flag",
+    "feature_flag": "es.query_rules_feature_flag_enabled",
+    "headers": {
+      "accept": [
+        "application/json"
+      ]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/_query_rules",
+          "methods": [
+            "GET"
+          ]
+        }
+      ]
+    },
+    "params": {
+      "from": {
+        "type": "int",
+        "description": "Starting offset (default: 0)"
+      },
+      "size": {
+        "type": "int",
+        "description": "specifies a max number of results to get (default: 100)"
+      }
+    }
```

You can also include a default param:
```suggestion
    "params": {
      "from": {
        "type": "int",
        "default": 0,
        "description": "Starting offset (default: 0)"
      },
      "size": {
        "type": "int",
        "default": 100,
        "description": "specifies a max number of results to get (default: 100)"
      }
    }
```

---

Thanks! Added 👍